graylog2-server/src/main/java/org/graylog2/indexer/indices/Indices.java

```diff
@@ -298,15 +298,29 @@ public long numberOfMessages(String indexName) throws IndexNotFoundException {
 
     public boolean exists(String indexName) {
```

Comment:
Please add tests for the following cases:
* Index exists
* Index does not exist

Ideally, the test for an alias would be separate, but it's also okay as a part of `IndicesTest#testAliasExists()`.